ui/src/app/components/resource-card.tsx

```diff
@@ -161,15 +161,15 @@ export class ResourceCard extends React.Component<Props, State> {
     return this.resourceType === ResourceType.DATA_SET;
   }
 
-  get actionsDisabled(): boolean {
-    return !this.writePermission;
-  }
-
-  get writePermission(): boolean {
+  get writerPermission(): boolean {
     return this.props.resourceCard.permission === 'OWNER'
```

Comment:
nit/idea: it's possible to implement permissions so that they support greater than, for example if you wanted owner to always be higher than (and include) writer. Then you could have `permission >= 'WRITER'`. Also, can we use an enum instead of string literals? I thought Swagger could define that for us from the API deinition.

Reply:
But does it expose a string-valued property like `WorkspaceAccessLevel.OWNER.name` or something so you don't have to spell it out?

I wouldn't rely on the existing numerical value for comparison unless it was explicitly designed that way b/c it's not future-proof.

Reply:
Yeah, in general I tend towards something like `this.props.resourceCard.permission === WorkspaceAccessLevel.OWNER` and pass around the access levels. (even over using a string value property)